plugins/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@
 #  Boston, MA 02110-1301, USA.
 
 set(PLUGINS
+	generic_plugin
```

Comment:
Please add the plugin to Makefile as well. We're still using it.

Reply:
Done.